providers/dns/joker/joker.toml

```
@@ -22,6 +22,25 @@ JOKER_API_KEY=<your API key> \
 lego --email myemail@example.com --dns joker --domains my.example.org run
 '''
 
+Additional = '''
+## SVC mode
+
+In the SVC mode, username and passsword are not my email and account passwords, but those displayed in Joker.com domain dashboard when enabling Dynamic DNS.
+
+Reference: https://joker.com/faq/content/6/496/en/let_s-encrypt-support.html
+
+1. please login at Joker.com, visit 'My Domains',
+   find the domain you want to add  Let's Encrypt certificate for, and chose "DNS" in the menu
+
+2. on the top right, you will find the setting for 'Dynamic DNS'.
+   If not already active, please activate it.
+   It will not affect any other already existing DNS records of this domain.
+
+3. please take a note of the credentials which are now shown as 'Dynamic DNS Authentication', consisting of a 'username' and a 'password'.
+
+4. this is all you have to do here - and only once per domain.
```

Copying text verbatim from other (proprietary!) sites is not a good style. Better to mark it as a citation:

```suggestion
As per [Joker.com documentation](https://joker.com/faq/content/6/496/en/let_s-encrypt-support.html):

> 1. please login at Joker.com, visit 'My Domains',
>    find the domain you want to add  Let's Encrypt certificate for, and chose "DNS" in the menu
>
> 2. on the top right, you will find the setting for 'Dynamic DNS'.
>    If not already active, please activate it.
>    It will not affect any other already existing DNS records of this domain.
>
> 3. please take a note of the credentials which are now shown as 'Dynamic DNS Authentication', consisting of a 'username' and a 'password'.
>
> 4. this is all you have to do here - and only once per domain.
```